lib/smb2/packet/ioctl_request.rb

```diff
@@ -65,5 +65,10 @@ class IoctlRequest < Smb2::Packet
       SMB2_0_IOCTL_IS_FSCTL: 1
     }.freeze
 
+    # @return [Symbol] a key in {Smb2::COMMANDS}
+    def self.command
```

Comment:
Unnecessary if you move to [class-constant lookup](https://github.com/rapid7/smb2/pull/28/files#r28820364)